### PR TITLE
updated ReadMe for Lucidum App

### PR DESCRIPTION
**Pull request by JKashyap96**
# PR Details

## Checklist

- [ ] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [ ] Ran unit tests locally.
- [ ] Updated screenshots/sample logs if any.

### CODE REVIEW

File: Lucidum/README.md [hunk 1]
```diff
@@ -30,8 +30,18 @@ _sourceCategory = Lucidum
...
 ## Collect Logs for Lucidum
 
+Use the Collection page to manage all of your Collectors and Sources. To access the Collection page, go to Manage Data > Collection > Collection.
+- In Sumo Logic, configure a [hosted collector](https://help.sumologic.com/03Send-Data/Hosted-Collectors)
+    and associate an [HTTP Logs and Metrics Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source#configure-an-http%C2%A0logs-and-metrics-source) with the collector. Copy the HTTP source URL.
+- For Lucidum Setup,
+    - create a new integration and use the HTTP source URL for the webhook_url:
+    ![Alt text](resources/screenshots/Lucidum_Integration.png?raw=true)
+    - create new scheduled action using the Sumo Integration:
+        - choose schedule, filter, output fields, Sumo configuration name and the Save and Run.
```
Comment (himanshu219) [inline Lucidum/README.md:40]: Need to add further screenshots from the pdf

File: bin/scr_app_list.txt [hunk 2]
```diff
@@ -1,4 +1,2 @@
 // This file contains a list of apps need to be updated in some SCR
-Lucidum/Lucidum.json:Lucidum/Lucidum.manifest.json
-Gigamon_HAWK/GigamonHawkApplicationMetadataIntelligence.json:Gigamon_HAWK/GigamonHawkApplicationMetadataIntelligence.manifest.json
-Gigamon_ThreatINSIGHT/GigamonHawkThreatINSIGHT.json:Gigamon_ThreatINSIGHT/GigamonHawkThreatINSIGHT.manifest.json
\ No newline at end of file
+Lucidum/Lucidum.json:Lucidum/Lucidum.manifest.json
```
Comment (himanshu219) [inline bin/scr_app_list.txt:2]: why do we need to include this in scr app list ? we won't need to deploy the app for updating docs.

**Review by himanshu219 — CHANGES_REQUESTED** [review]

File: bin/scr_app_list.txt (hunk 2 above)
Comment (JKashyap96) [reply bin/scr_app_list.txt:2]: Not required.

File: Lucidum/README.md (hunk 1 above)
Comment (JKashyap96) [reply Lucidum/README.md:40]: Have added more screenshots, please review.

**Review by himanshu219 — APPROVED** [review]
LGTM

**Pull request merged**